ios/Podfile.lock

```diff
@@ -2608,12 +2608,12 @@ SPEC CHECKSUMS:
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   SwiftProtobuf: 7773c4e96a99d7b8ab7cda0fc30a883732ff93b1
   TokenCore: 3235fc5e40d5364f169618de8ba37929607618ff
-  TokenStateReactNativeLib: 7d0ff81bceaa26361ad7fc5e90cf161eb1a4b921
+  TokenStateReactNativeLib: 02b5534c9a2a823123fc6cc2ce95fe3b52f0b82f
   Turf: aa2ede4298009639d10db36aba1a7ebaad072a5e
   VisualCodeModeliOSLib: 826875ce13fe87af1517d5d730a23181ca3b21be
   VisualCodeViewiOSLib: 6f3ee3110493f2d6b2f9d7d093acc75a098bd948
   Yoga: c716aea2ee01df6258550c7505fa61b248145ced
 
 PODFILE CHECKSUM: 1379ffa3dfb4fef08c9b1973f83c992e336715c2
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
We should make a separate PR for upgrading this version, so the changes in this file should be removed from this PR